geonode_dominode/geonode_dominode/urls.py

```diff
@@ -35,10 +34,6 @@
         name='home'),
     url(r'^groups/group/(?P<slug>[-\w]+)/$',
         GroupDetailView.as_view(), name='group_detail'),
-    url(r'^cli/group/(?P<slug>[-\w]+)/$',
-        CLIListView.as_view(), name='cli_view'),
-    url(r'^cli/execute/(?P<cli_slug>[-\w]+)/',
-        cli_executor, name='cli_execute'),
     url(r'^cli/sync_geoserver/',
```

Comment:
please drop this `cli` prefix

Reply:
Do you want to change into other namespace?

Reply:
I just don't want there to be reference to `cli` anywhere if possible. This is misleading since we are not using any sort of CLI with the current implementation